src/Helpers/constants.js

```diff
@@ -864,6 +891,8 @@ export const DEFAULT_FILTER_DATA = {
     rhel_version: []
 };
 
+export const DEFAULT_SORTER_DATA = '-cvss_score';
```

Comment:
It might be a good idea to name it more appropriately, for example `CUSTOM_REPORT_SORT_OPTIONS` to avoid confusion

Reply:
the example was already used for renaming `SORT_CVE_OPTIONS`. Does `DEFAULT_CUSTOM_REPORT_SORTER_DATA` sound good?

Reply:
We have similar component for `Publish date`, basically same, it just contains radio buttons to indicate selection. It does not close on its own when selection is made. Do we want to add "on select close" functionality there too?

Reply:
`CUSTOM_REPORT_DEFAULT_SORT` sounds good `DEFAULT_CUSTOM_REPORT_SORTER_DATA` sounds like word "default" binds to custom report which is not the case and if it's just a string there is not a lot of "data" 😀.

That is a really good question, it's a question of button clicking feedback - if the user clicks radio button and the radio button changes that is a good feedback. If there a no radio buttons menu closing is a good feedback. So I think the other dropdowns with radio buttons are good as they are, but it does make sense to make this sort menu also use radio items for consistency, doesn't it? I would just leave it as is (with closing after item select) and add a comment on the mockup here https://marvelapp.com/prototype/gb738c4/screen/80582731 if it would make sense to make items have radio buttons for consistency with other dropdowns in that modal.

Reply:
Well user can see that in first row was value changes so there is a kind of feedback, but sure I will as you say, it makes sense